.github/workflows/clp-core-build.yaml

```diff
@@ -0,0 +1,48 @@
+name: Test Building CLP Core on Ubuntu Focal, Bionic and Centos 7.4
+
+on:
+  push:
+    branches: [ main ]
+    paths:
+      - 'components/core/**'
+      - '.github/workflows/clp-core-build.yaml'
```

Comment:
Should it also depend on `.github/workflows/clp-dependency-image-build.yaml`?

Reply:
Not really, the dependency between the workflows will be added via the workflow trigger action (in the next PR)